tests/unit/src/test/scala/tests/NewFilesLspSuite.scala

```diff
@@ -155,26 +132,64 @@ class NewFilesLspSuite extends BaseLspSuite("new-files") {
                                 |  "a": { }
                                 |}
                                 |""".stripMargin)
-      _ <- server.executeCommand(
-        ServerCommands.NewScalaFile.id,
-        MetalsNewScalaFileParams(
-          workspace.resolve("a/src/main/scala/foo").toURI.toString,
-          "",
-          "package-object"
-        )
-      )
-      _ = assert(
-        workspace.resolve("a/src/main/scala/foo/package.scala").exists
-      )
-      _ <- server.didSave("a/src/main/scala/foo/package.scala")(identity)
-      _ = assertNoDiff(
-        workspace.resolve("a/src/main/scala/foo/package.scala").readText,
+      _ <- check(
+        workspace.resolve("a/src/main/scala/foo").toURI.toString(),
+        "package-object",
+        name = None,
+        workspace.resolve("a/src/main/scala/foo/package.scala"),
         """|package object foo {
            |  
            |}
            |""".stripMargin
       )
+    } yield ()
+  }
 
+  private def check(
```

Comment:
I like the direction, but I thought of something similar to RenameLspSuite and other, where we would even invoke `test` from here.

Reply:
done